core/domain/user_jobs_one_off.py

```diff
@@ -87,6 +87,29 @@ def reduce(key, stringified_username_counter):
         yield (key, len(username_counter))
 
 
+class UserBioLengthOneOffJob(jobs.BaseMapReduceJobManager):
```

Comment:
LongUserBiosOneOffJob perhaps, since you're only checking for len > 500. I think that conveys the purpose of the job a bit better.